### DISCUSSION TITLE
Zooming out should keep list in the visual range

### DISCUSSION THREAD

**Discussion by mafsi** [vslinko/obsidian-zoom#1] [Archive]
**Describe the bug**
On a long note Zooming out from a list shows the beginning of the page.

**Expected behavior**
On a long note Zooming out from a list should keep the page where it is.

**Screenshots**
![zoom_out](https://user-images.githubusercontent.com/61430691/114501475-bc840a80-9c32-11eb-9a18-701c28528916.gif)

**Desktop (please complete the following information):**
 - Windows 10
 - Obsidian Version: 0.11.13

**Comment by vslinko** [comment]
@mafsi Thank you for your contribution!

In my opinion, this looks more like an unimplemented feature rather than a bug. Also, unfortunately, it is not so easy to implement correctly.
Moving the issue into discussions for a while.

**Comment by vslinko** [comment]
Discussion was transferred from https://github.com/vslinko/obsidian-outliner to https://github.com/vslinko/obsidian-zoom

**Comment by rapatel0** [comment]
One note to people with this issue. The cursor actually does rebound to the right place in the document. so you just need to hit the arrow key to pop to the right visual range.

Also it might be a stupid hack to just hit up down on the arrow key to reposition the editor. Not sure if the api will support that though

**Comment by vslinko** [comment]
Should be fixed in https://github.com/vslinko/obsidian-zoom/releases/tag/0.2.4
